### ISSUE TITLE
Search box shoud have focus on directory search and not create a new file dialog

### ISSUE THREAD

**Issue by cyberwombat** [atom/atom#4474]
When one right clicks on a folder and select search directory the keyboard focus should be in the search box. It used to be so a few versions ago but stopped a few months ago for unknown reasons. It makes it a bit of a pain as the tendency is to type right after selecting search but this now causes a new file  dialog to open instead. Current version is 0.153

**Comment by jmargolisvt** [comment]
Why was this closed?  This bug still exists in v1.16.0.

**Comment by 50Wliu** [comment]
@jmargolisvt Please open a new issue.
